README.md

```diff
@@ -19,172 +19,62 @@ by its developers, nor is it "supported" software.
 
 ## Quick Start
 
-### Setup
+Try out ratify in Kuberenetes through Gatekeeper as the admission controller.
 
-- Build the `ratify` binary
+- Setup Gatekeeper with [external data](https://github.com/open-policy-agent/gatekeeper/pull/1677)
 
 ```bash
-git clone https://github.com/deislabs/ratify.git
-go build -o ~/bin ./cmd/ratify
-```
-
-- Build the `ratify` plugins and install them in the home directory
-
-```bash
-go build -o ~/.ratify/plugins/ ./plugins/verifier/sbom
-```
+helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts
 
-- `ratify` is ready to use
-
-```bash
-$ ratify --help
-Ratify is a reference artifact tool for managing and verifying reference artifacts
-
-Usage:
-  ratify [flags]
-  ratify [command]
-
-Available Commands:
-  completion  generate the autocompletion script for the specified shell
-  discover    Discover referrers for a subject
-  resolve     Resolve digest of a subject that is referenced by a tag
-  help        Help about any command
-  referrer    Discover referrers for a subject
-  serve       Run ratify as a server
-  verify      Verify a subject
-
-Flags:
-  -h, --help   help for ratify
-
-Use "ratify [command] --help" for more information about a command.
+helm install gatekeeper/gatekeeper  \
+    --name-template=gatekeeper \
+    --namespace gatekeeper-system --create-namespace \
+    --set enableExternalData=true \
+    --set controllerManager.dnsPolicy=ClusterFirst,audit.dnsPolicy=ClusterFirst
 ```
 
-### Verify a Graph of Supply Chain Content
-
-To get started with `ratify`, follow the step below.
-
-- Create a graph of Supply Chain Content
-- Discover the graph using `ratify`
-- Verify the graph using `ratify`
-
-This section outlines instructions for each of the above steps.
-
-#### **Create a graph of Supply Chain Content**
-
-A graph of supply chain content can be created with different tools that can manage individual supply chain objects within the graph. For this quick start, the steps outlined in [Notary V2 project](https://notaryproject.dev/blog/2021/announcing-notation-alpha1/) will be used to create a sample graph with [`notation`](https://github.com/notaryproject/notation) and [`ORAS`](https://github.com/oras-project/oras/releases/tag/v0.2.1-alpha.1) CLI.
-
-- Run a local instance of the [CNCF Distribution Registry](https://github.com/oras-project/distribution), with [ORAS Artifacts](https://github.com/oras-project/artifacts-spec/blob/main/artifact-manifest.md) support.
+- Deploy ratify and a `demo` constraint on gatekeeper
 
 ```bash
-export PORT=5000
-export REGISTRY=localhost:${PORT}
-export REPO=net-monitor
-export IMAGE=${REGISTRY}/${REPO}:v1
-
-docker run -d -p ${PORT}:5000 ghcr.io/oras-project/registry:v0.0.3-alpha
-```
-
-- Build & Push an image
+git clone https://github.com/deislabs/ratify.git
```

Comment:
instead of cloning the repo, can we do "kubectl apply -f <url>?

Reply:
@etrexel - Do you think you can help simply this and remove the clone step for the demo?

Reply:
https://github.com/deislabs/ratify/issues/100